server/mailet/mailetcontainer-camel/src/test/java/org/apache/james/mailetcontainer/lib/AbstractStateMailetProcessorTest.java

```diff
@@ -201,7 +202,7 @@ public void afterMailet(Mailet m, String mailName, String state, long processTim
         } catch (MessagingException e) {
             catched = true;
         }
-        assertTrue(catched);
+        assertFalse(catched);
```

Comment:
```
boolean catched = false;
try {
    // do something
} catch(Exception e) {
    catched = true;
}
assertFalse(catched);
```

Is strictly equivalent to : 

```
// do something
```

(and the second options prints error details, easing debugging...)

Reply:
Okay, I will change that.